app/commands/submission/create.rb

```diff
@@ -70,35 +71,25 @@ def init_services
           Submission::TestRun::Init.(submission_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
         end
       ]
-      if major
-        jobs += [
-          Thread.new do
-            Submission::Analysis::Init.(submission_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
-          end,
-          Thread.new do
-            Submission::Representation::Init.(submission_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
-          end
-        ]
-      end
+      # TODO: Move to iteration create
+      # jobs += [
+      #   Thread.new do
+      #     Submission::Analysis::Init.(submission_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
+      #   end,
+      #   Thread.new do
+      #     Submission::Representation::Init.(submission_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
+      #   end
+      # ]
```

Comment:
This means temporarily there won't be any analyses or representations being run?

Reply:
Correct. But only for a day or two.